domain/src/main/kotlin/no/nav/su/se/bakover/domain/visitor/LagBrevRequestVisitor.kt

```diff
@@ -329,6 +334,7 @@ class LagBrevRequestVisitor(
         beregning = beregning,
         fritekst = fritekst,
         harEktefelle = harEktefelle,
+        forventetInntektStørreEnn0 = uføregrunnlag.sumOf { it.forventetInntekt } > 0,
```

Comment:
@jacob-meidell Når jeg tenker meg om fungerer ikke dette likt som Beregning, der fradragene er periodisert. Men det går absolutt an å lage en funksjon på wrapper-typen som deler opp i måneder og summerer forventetInntekt for hele året?

Reply:
Litt usikker på hva som er behovet her, men dersom det er tilstrekkelig å vise denne teksten 1 gang i brevet for alle tilfeller hvor det eksisterer minst en måned med IEU > 0 vil vel den summeringen der fungere greit nok.  Dersom visningen av teksten skal variere med periodene i beregningen vil det sannsynligvis ikke fungere som tiltenkt.